sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py

```diff
@@ -48,12 +49,13 @@
 NEW_TYPES_OUTPUT_SCHEMA = (
     '{"fields": [{"name": "bytes","type": "BYTES"},'
     '{"name": "date","type": "DATE"},{"name": "time","type": "TIME"}]}')
-NEW_TYPES_OUTPUT_VERIFY_QUERY = ('SELECT date FROM `%s`;')
-# There are problems with query time and bytes with current version of bigquery.
+NEW_TYPES_OUTPUT_VERIFY_QUERY = ('SELECT bytes, date, time FROM `%s`;')
 NEW_TYPES_OUTPUT_EXPECTED = [
-    (datetime.date(2000, 1, 1),),
-    (datetime.date(2011, 1, 1),),
-    (datetime.date(3000, 12, 31),)]
+    (b'xyw=', datetime.date(2011, 1, 1), datetime.time(23, 59, 59, 999999),),
```

Comment:
Let's remove the `=` sign after `xyw` and others here to avoid confusion that these strings need to be base64-encoded (odd-length base64 encoded strings are padded with `=`).
Also, let's replace one of the usecases with a utf-8-decodable byte-sequence. This use-case will help to make sure that this sequence doesn't get accidentally decoded somewhere. For example, we can use `b'\xe4\xbd\xa0\xe5\xa5\xbd'`.